src/t_stream.c

```diff
@@ -2153,13 +2153,13 @@ void xlenCommand(client *c) {
     addReplyLongLong(c,s->length);
 }
 
-/* XREAD [BLOCK <milliseconds>] [COUNT <count>] STREAMS key_1 key_2 ... key_N
- *       ID_1 ID_2 ... ID_N
+/* XREAD [COUNT <count>] [BLOCK <milliseconds>] STREAMS key [key...] id [id...] 
```

Comment:
the order among optional args does't matter. the fact it was changed in the docs, doesn't mean we need to modify the source code.